sway/ipc-server.c

```diff
@@ -480,6 +484,7 @@ void ipc_client_handle_command(struct ipc_client *client) {
 		const char *json = cmd_results_to_json(results);
 		char reply[256];
 		int length = snprintf(reply, sizeof(reply), "%s", json);
+		free((char *)json);
```

Comment:
ah, this is actually not correct; now that I've found the doc of json_object_to_json_string it says json_object_put should free it but that's no longer available in this scope.
I guess we should strdup this

Reply:
Done that.